### PR TITLE
[FIX] Sequence number for ' Room types  and Miscellaneous' 

### PR DESCRIPTION
**Pull request by jananjoy**

### CODE REVIEW

File: project_resource_calendar/views/miscellaneous_view.xml [hunk 1]
```diff
@@ -28,14 +28,21 @@
...
-                            </h1>
-                        </div>
+                        <field name="name"/>
+                        <field name="sequence"/>
                     </group>
                 </sheet>
             </form>
         </field>
     </record>
+    <record id="resource_calendar_miscellaneous_tree" model="ir.ui.view">
```
Comment (rimbendhaou) [inline project_resource_calendar/views/miscellaneous_view.xml]: we already have a tree, just modify it

File: project_resource_calendar/views/room_type_view.xml [hunk 2]
```diff
@@ -31,13 +31,22 @@
...
-                    </div>
+                    <group>
+                        <field name="name"/>
+                        <field name="sequence"/>
+                   </group>
                 </sheet>
             </form>
         </field>
     </record>
+    <record id="room_type_tree" model="ir.ui.view">
```
Comment (rimbendhaou) [inline project_resource_calendar/views/room_type_view.xml]: same comment (tree already exist)

File: project_resource_calendar/models/room_type.py [hunk 3]
```diff
@@ -9,3 +9,6 @@ class RoomType(models.Model):
     name = fields.Char(
         string='Room Type'
     )
+    sequence = fields.Integer(
```
Comment (rimbendhaou) [inline project_resource_calendar/models/room_type.py:13]: add order by sequence

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

**Review by rimbendhaou — APPROVED** [review]

**Review by rimbendhaou — COMMENTED** [review]
translation

**Review by rimbendhaou — CHANGES_REQUESTED** [review]
add translation for Sequence--> Séquence

File: project_resource_calendar/models/miscellaneous.py [hunk 4]
```diff
@@ -5,7 +5,11 @@
 
 class Miscellaneous(models.Model):
```
Comment (eilst) [inline project_resource_calendar/models/miscellaneous.py:6]: @jananjoy I think we should remove this model, it's not used anywhere, right?

**Comment by eilst** [comment]
LGTM

**Review by eilst — APPROVED** [review]

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**